lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.h

```diff
@@ -6,13 +6,20 @@
 #ifdef ENABLE_JS_BUILTINS
 namespace Js
 {
+
+    #define enum_file(class, type, obj) class##_##type,
+    typedef enum {
+        JsBuiltIns(enum_file)
+    } JsBuiltInFile;
+    #undef enum_file
```

Comment:
Enum for selecting the different js files - using this mechanism adding more JS files with the correct names will extend this automatically.